lib/phoenix_live_view/channel.ex

```diff
@@ -629,8 +629,9 @@ defmodule Phoenix.LiveView.Channel do
     Process.monitor(transport_pid)
     load_csrf_token(endpoint, socket_session)
 
+    url = Map.fetch!(params, "url")
```

Comment:
Does the client always submits the URL? Even for widget-style LiveViews?

Reply:
yes